testgen/__main__.py

```diff
@@ -727,6 +729,8 @@ def run(debug: bool):
                 app_file,
                 "--ui.hideSidebarNav=true",
                 "--browser.gatherUsageStats=false",
+                f"{'--server.sslCertFile=' + settings.SSL_CERT_FILE if use_ssl else ''}",
+                f"{'--server.sslKeyFile=' + settings.SSL_KEY_FILE if use_ssl else ''}",
```

Comment:
```suggestion
                f"--server.sslCertFile={settings.SSL_CERT_FILE}" if use_ssl else "",
                f"--server.sslKeyFile={settings.SSL_KEY_FILE}" if use_ssl else "",
```

Looks more readable =)

Reply:
Oh, that's much better - thanks!